src/test/components/StackChart.test.js

```diff
@@ -48,154 +53,162 @@ const GRAPH_WIDTH =
   GRAPH_BASE_WIDTH + TIMELINE_MARGIN_LEFT + TIMELINE_MARGIN_RIGHT;
 const GRAPH_HEIGHT = 300;
 
-describe('StackChart', function() {
+xdescribe('StackChart with samples', function() {
```

Comment:
I didn't review the tests since they are skipped, but they will need to be completed in order to merge the PR. Do you want help here to figure out the proper approach?